Add GraphQL tool Swan Lake 2201.1.0 release note (#1952)
ballerina-platform/ballerina-release: release-notes/swan-lake-2201.1.0-release-note.md

#### New Features

##### GraphQL Tool

> - Introduced the Ballerina GraphQL tool that will make it easy for you to generate a client in Ballerina given the GraphQL schema (SDL) and GraphQL queries. Ballerina Swan Lake supports the GraphQL specification [October 2021 edition](https://spec.graphql.org/October2021/). You can find more details about the tool [here](https://github.com/ballerina-platform/graphql-tools/blob/main/README.md).

Reviewer: ```suggestion
- Introduced the Ballerina GraphQL tool, which will make it easy for you to generate a client in Ballerina given the GraphQL schema (SDL) and GraphQL queries. Ballerina Swan Lake supports the GraphQL specification [October 2021 edition](https://spec.graphql.org/October2021/). For more details, see [graphql-tools](https://github.com/ballerina-platform/graphql-tools/blob/main/README.md).
```

Author: Update the PR with the suggested change. Thank you.
